common/src/main/java/com/thoughtworks/go/remote/adapter/RuntimeTypeAdapterFactory.java

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright 2020 ThoughtWorks, Inc.
```

Comment:
I think you should get the full original file, _along_ with its license and then add this license on top of this, with a comment explaining where it came from.

Reply:
Will do.